lib/core-extensions/UserCSS/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+	"name": "UserCSS",
```

Comment:
I do not really mind. For me something with CSS in the name would be most obvious, such as "Custom CSS"

Reply:
I like CSS Tweaks and JS Tweaks, but User CSS / UserCSS and User JS / UserJS are decent enough names.

Reply:
I don't mind if it is not named "UserCSS", but I did not put a space because this name will be used for the configuration file directory name.
I don't want to include spaces in any file paths.

Although the specifications will be changed, could we use `$this->getEntrypoint` instead of `$this->getName()` for the configuration file directory name?

Reply:
@hkcomori That is a reasonable suggestion, thanks

Reply:
Some AI suggestions:

- CSS Customizer
- CSS Personalizer
- Flexible CSS Tuner
- CSS Overwriter
- CSS Remixer

Reply:
Those suggestions are worse to much worse imho. :-)

Reply:
OK, I changed the directory name to entrypoint and added a space to the name.